sqlstore/src/main/resources/db/migration/V0028__model_refactor.sql

```diff
@@ -2,3 +2,28 @@ ALTER TABLE SampleAdditionalInfo ADD COLUMN groupId int(10);
 ALTER TABLE SampleAdditionalInfo ADD COLUMN groupDescription varchar(255);
 
 UPDATE SampleAdditionalInfo SET groupId = (SELECT groupId FROM SampleAnalyte WHERE SampleAdditionalInfo.sampleId = SampleAnalyte.sampleId), groupDescription = (SELECT groupDescription FROM SampleAnalyte WHERE SampleAdditionalInfo.sampleId = SampleAnalyte.sampleId);
+
+ALTER TABLE SampleAdditionalInfo DROP FOREIGN KEY `FKeutn2473w3yr16khgalspuviw`; -- createdBy fkey
+ALTER TABLE SampleAdditionalInfo DROP FOREIGN KEY `FKp8bvx3e7jsmnyw51toi7mq7cq`; -- updatedBy fkey
```

Comment:
The FK names are declared in the migration script, so they are named consistently (at least the ones in question). They mostly are randomly generated and not nice names, but that's not a problem. I'll fix the comments